vividus-tests/src/main/resources/data/tables/system/mobile_app/locators/android.table

```diff
@@ -1,19 +1,21 @@
-|textFieldValueAttribute    |text                                                                     |
-|scrollableMenuXpath        |//android.widget.ScrollView                                              |
-|qrCodeContainerXpath       |//android.view.ViewGroup[@resource-id='RNE__Image__children__container'] |
-|firstItemAccessibilityId   |Item 1                                                                   |
-|secondItemAccessibilityId  |Item 2                                                                   |
+|textElementXpath           |//android.widget.TextView                                                              |
+|menuButtonXpath            |//android.widget.TextView[@text='Button']                                              |
+|menuInputXpath             |//android.widget.TextView[@text='Input']                                               |
+|nameDisplayXpath           |//android.widget.TextView[@text='${text}']                                             |
+|menuQrCodeXpath            |//android.widget.TextView[@text='QR Code']                                             |
+|menuWaitXpath              |//android.widget.TextView[@text='Wait']                                                |
+|menuScrollViewXpath        |//android.widget.TextView[@text='Scroll View']                                         |
+|menuSliderXpath            |//android.widget.TextView[@text='Slider']                                              |
+|nameInputXpath             |//android.widget.EditText[@text='Enter your name...']                                  |
+|action                     |COMPARE_AGAINST                                                                        |
+|scrollViewXpath            |//android.widget.ScrollView                                                            |
+|carouselViewXpath          |//android.widget.TextView[@text='Carousel']                                            |
+|swipeableAreaXpath         |//android.widget.HorizontalScrollView                                                  |
+|textFieldValueAttribute    |text                                                                                   |
+|scrollableMenuXpath        |//android.widget.ScrollView                                                            |
+|qrCodeContainerXpath       |//android.view.ViewGroup[@resource-id='RNE__Image__children__container']               |
+|firstItemAccessibilityId   |Item 1                                                                                 |
+|secondItemAccessibilityId  |Item 2                                                                                 |
+|secondVisibleItemXpath     |//android.widget.HorizontalScrollView/android.view.ViewGroup/android.view.ViewGroup[2] |
+|firstVisibleItemXpath      |//android.widget.HorizontalScrollView/android.view.ViewGroup/android.view.ViewGroup[1] |
```

Comment:
can we use something better than index-based xpaths?

Reply:
There is not much choice

<img width="1259" alt="android" src="https://user-images.githubusercontent.com/18533593/198702951-d0718617-f28e-4677-93a9-1d847be51a23.png">